cluster-install/install.sh

```diff
@@ -17,11 +17,13 @@ _KUBERNETES_VERSION=""
 # existing Trident information (if any) has been collected.
 _TRIDENT_COLLECTION_STEP_CALLED="false"
 _EXISTING_TORC_NAME="" # TORC is short for TridentOrchestrator (works with kubectl too)
+_EXISTING_TORC_PULL_SECRETS="" # Newline-delimited values, e.g. 'secret1\nsecret2\nsecret3'
 _EXISTING_TRIDENT_NAMESPACE=""
 _EXISTING_TRIDENT_IMAGE=""
 _EXISTING_TRIDENT_ACP_ENABLED=""
 _EXISTING_TRIDENT_ACP_IMAGE=""
 _EXISTING_TRIDENT_OPERATOR_IMAGE=""
+_EXISTING_TRIDENT_OPERATOR_PULL_SECRETS="" # Newline-delimited values, e.g. 'secret1\nsecret2\nsecret3'
```

Comment:
These two will hold the existing pull secrets, which we then use when crafting corresponding upgrade patches to determine whether we should add the current IMAGE_PULL_SECRET or not.